lunes_cms/cms/models/feedback.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+class Feedback(models.Model):
+    """
+    Contains a title, a description and a reference to the matter of feedback
+    """
+
+    id = models.AutoField(primary_key=True)
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    MATTER_CHOICES = (
+        ('general','GENERAL'), 
+        ('training_set','TRANING_SET'), 
+        ('document','DOCUMENT'), 
+        ('user_group','USER_GROUP')
+    )
+    matter = models.ForeignKey(max_length=4, choices=MATTER_CHOICES, default='general')
```

Comment:
```suggestion
    matter = models.CharField(
        max_length=12, choices=MATTER_CHOICES, default=MATTER_CHOICES[0][0]
    )
```
A `ForeignKey` is a link to another model. If you want to store constant strings in this field, you need a `CharField`.
Also, it makse sense not to hardcode the default value, but to reference the choice definition.